docs/userguide/dockerizing.md

```diff
@@ -57,6 +57,9 @@ So what happened to our container after that? Well Docker containers
 only run as long as the command you specify is active. Here, as soon as
 `Hello world` was echoed, the container stopped.
 
+  <img src="/userguide/hello_world_images/helloworld.gif">
```

Comment:
Could you change the images to use standard Markdown notation? i.e.;

```Markdown
![hello-world](/userguide/hello_world_images/helloworld.gif)
```

After changing, it's good practice to check if it works correctly, by running `make docs` from the root of your repository, as explained here: https://github.com/docker/docker/blob/master/docs/README.md